code/drasil-docLang/Drasil/DocumentLanguage/TraceabilityGraph.hs

```diff
@@ -91,24 +73,37 @@ checkUID t si
   where s = _sysinfodb si
 
 -- | Similar to 'checkUID' but prepends domain for labelling.
-checkUID' :: UID -> SystemInformation -> String
-checkUID' t si
-  | Just _ <- Map.lookupIndex t (s ^. dataDefnTable)        =  "DD: " ++ t --Directly encoded since these are unlikely to change
-  | Just _ <- Map.lookupIndex t (s ^. insmodelTable)        =  "IM: " ++ t
-  | Just _ <- Map.lookupIndex t (s ^. gendefTable)          =  "GD: " ++ t
-  | Just _ <- Map.lookupIndex t (s ^. theoryModelTable)     =  "TM: " ++ t
-  -- Concept instances can range from likely changes to non-functional requirements, so use domain abbreviations for labelling.
-  | Just _ <- Map.lookupIndex t (s ^. conceptinsTable)      = (safeFromJust $ getA $ defResolve s $ sDom $ cdom $ conceptinsLookup  t $ s ^. conceptinsTable) ++ ": " ++ t
-  | Just _ <- Map.lookupIndex t (s ^. sectionTable)         = "Sec: " ++ t
-  | Just _ <- Map.lookupIndex t (s ^. labelledcontentTable) = "Content: " ++ t
+checkUIDAbbrev :: SystemInformation -> UID -> String
+checkUIDAbbrev si t
+  | Just _ <- Map.lookupIndex t (s ^. dataDefnTable)        = abrv $ datadefnLookup    t (s ^. dataDefnTable)
```

Comment:
why are you throwing away the index that you just got? In fact, why not look up the value, instead of doing it twice?